src/containers/Tenant/Query/QueryResult/components/ResultSetsViewer/ResultSetsViewer.tsx

```diff
@@ -23,58 +22,54 @@ interface ResultSetsViewerProps {
+        const tabsItems: TabsItemProps[] =
+            resultSets?.map((_, index) => {
+                const resultSet = resultSets?.[index];
+                const rowsPerSecond = resultSet?.streamMetrics?.rowsPerSecond;
+                const rowsPerSecondFormatted =
+                    rowsPerSecond && rowsPerSecond > 1000
+                        ? `${(rowsPerSecond / 1000).toFixed(0)}k`
+                        : rowsPerSecond?.toFixed(0);
+                return {
+                    id: String(index),
+                    title: (
+                        <div className={b('tab-title')}>
+                            <Text>
+                                {`Result${resultSets.length > 1 ? ` #${index + 1}` : ''}${resultSets?.[index]?.truncated ? '(T)' : ''}`}
+                            </Text>
+                            <Text color="secondary">{resultSet.result?.length || 0}</Text>
+                        </div>
+                    ),
+                    label: rowsPerSecondFormatted
+                        ? {
+                              content: `${rowsPerSecondFormatted} rows/s`,
+                          }
+                        : undefined,
+                };
+            }) || [];
 
-            return (
-                <div>
-                    <Tabs
-                        className={b('tabs')}
-                        size="l"
-                        items={tabsItems}
-                        activeTab={String(selectedResultSet)}
-                        onSelectTab={(tabId) => setSelectedResultSet(Number(tabId))}
-                    />
-                </div>
-            );
-        }
-
-        return null;
-    };
-
-    const renderResultHeadWithCount = () => {
         return (
-            <div className={b('head')}>
-                <Text variant="subheader-3">
-                    {currentResult?.truncated ? i18n('title.truncated') : i18n('title.result')}
-                </Text>
-                {currentResult?.result ? (
-                    <Text color="secondary" variant="body-2" className={b('row-count')}>
-                        {`(${currentResult?.result.length}${
-                            currentResult.streamMetrics?.rowsPerSecond
-                                ? `, ${currentResult.streamMetrics.rowsPerSecond.toFixed(0)} rows/s`
-                                : ''
-                        })`}
-                    </Text>
-                ) : null}
+            <div>
+                <Tabs
+                    className={b('tabs')}
+                    size="l"
+                    items={tabsItems}
+                    activeTab={String(selectedResultSet)}
+                    onSelectTab={(tabId) => setSelectedResultSet(Number(tabId))}
+                />
             </div>
         );
     };
 
     return (
         <div className={b('result-wrapper')}>
-            {renderTabs()}
             {props.error ? <QueryResultError error={error} /> : null}
+            {renderTabs()}
```

Comment:
why tabs are below error?..

Reply:
<img width="357" alt="image" src="https://github.com/user-attachments/assets/d1c6ee8d-8154-47a3-8880-6f0401b6e768" />

because its the design

Reply:
https://www.figma.com/design/bNbecZuPFDDrJ7ghaXZ2Fe/Tickets?node-id=5461-76831&t=sLhJi7M9IpQH2qWj-0